nb2workflow/deploy.py

```diff
@@ -298,7 +300,37 @@ def _build_with_kaniko(git_origin,
         return container_metadata
 
 
-def _build_with_docker(git_origin, 
+def _extract_resource_requirements(local_repo_path):
+    from oda_api.ontology_helper import Ontology
+    from nb2workflow.nbadapter import NotebookAdapter
```

Comment:
Why not to put these imports to the top?

Reply:
done